control/models.py

```diff
@@ -161,6 +161,7 @@ class JobArtefact(models.Model):
     file = models.FileField(upload_to='artefacts/%Y/%m/%d/')
     artefact_type = models.CharField(max_length=4, default=JobArtefactType.SOURCE, choices=JobArtefactType.choices)
     mime_type = models.CharField(max_length=100, default='application/octet-stream')
+    order = models.IntegerField(default=0)
```

Comment:
The name is not descriptive, consider documenting it. IPP names this field `document-number`, consider using the same name here (instead of `order`). 

In my opinion this field should be required, but should not have a default value. The IPP Document Object standard requires starting the numbering from `1`. Adding a default also introduces an opportunity for the numbering to contain duplicates, which is also incompatible with the IPP Document Object standard.